Detectors/MUON/MCH/GlobalMapping/src/testChannelCode.cxx

```diff
@@ -42,6 +42,7 @@ BOOST_AUTO_TEST_CASE(Ctor1)
     o2::mch::ChannelCode(557, 7, 60)};
 
   for (const auto& id : ids) {
+    BOOST_CHECK_EQUAL(id.isValid(), true);
```

Comment:
you can profit from updating the tests to also check that getDePadIndex() returns the expected value (it's the only part of the channel identifier not tested)

Reply:
done